svelte/components/Nft/NftGrid.svelte

```diff
@@ -0,0 +1,96 @@
+<script lang="ts">
+  import type { NftType } from "lib/ktypes";
+
+  import { nftName, nftOpenSeaUrl, addressSame, getNetwork } from "lib/kconfig";
+
+  import { nftGetImageLink } from "lib/knft-get-metadata";
+
+  /////////////////////////////////////////////////
+  //  <NftGrid {chainId} {tokenID} {account} {nft} {platform}? />
+  // Display NFT card for Grid mode
+  /////////////////////////////////////////////////
+  export let chainId: number;
+  export let tokenID: string = "";
+  export let account: string = undefined;
+
+  export let nft: NftType;
+  export let index: number;
+
+  export let platform: string = "dapp";
+  ///////////////////////////////////////////////////////////////////////////////
+  const displayNftSolo = (evt: Event, clickedNftTokenID: string) => {
+    if (!(evt.target as HTMLInputElement).classList.contains("btn")) {
```

Comment:
use clickOutside action ? (or opposite ?)

Reply:
it's a filter for inside the component, not shure it will be interesting, have to talk about this